CHANGELOG.rst

```diff
@@ -17,4 +17,7 @@ netuitive-agent-elasticbeanstalk
 - Exclude device mapper virtual drives from DiskSpaceCollector by default
 - Add a BaseCollector config (on by default)
 
-NEXT
\ No newline at end of file
+NEXT
+
+- Add pkg concurrent-log-handler to fix locking, I/O errors when multiprocess logging to single file
```

Comment:
This note can be removed too since it was a change in one of the upstream projects right?

Reply:
Done